### DISCUSSION TITLE
支持设置读写操作的单独超时

### DISCUSSION THREAD

**Discussion by iwannay** [bfenetworks/bfe#1177] [General]
像nginx这类产品，读写超时一般只针对两个连续读取或写入操作之间的一段时间设置，而不是针对整个请求主体的传输。目前bfe的读写超时是针对的整个请求，所以造成大文件上传或下载超时中断，而nginx并不会超时，请问bfe后面是否会考虑增加这类特性？

**Comment by iyangsj** [comment]
@iwannay 感谢你的反馈！示例场景一般可通过调整合适的超时时间来避免中断正常下载，同时避免慢连接攻击。未来的版本考虑同时支持单次网络读/写操作的超时

**Comment by iwannay** [comment]
@iyangsj 好的，感谢

**Comment by jerrytanjunjie888** [comment]
好兴奋 又发现新大陆..
